package.json

```diff
@@ -3,11 +3,8 @@
   "version": "1.0.0",
   "description": "Loads environment variables from .env file",
   "main": "lib/main.js",
-  "directories": {
-    "test": "test"
-  },
   "scripts": {
-    "test": "gulp ci"
+    "test": "lab test/* --coverage && standard"
```

Comment:
what's got everyone sold on `standard`? no config?

beyond `eslint` being bundled, it also offers more rules, wider community support, and more [integrations](http://eslint.org/docs/integrations/). to use with `lab`, you just pass `-L` or `--lint`